exodus_lambda/functions/origin_request.py

```diff
@@ -171,28 +125,13 @@ def resolve_aliases(self, uri):
         return uri
 
     def handle_cookie_request(self, event):
-        now = datetime.utcnow()
-        ttl = timedelta(minutes=self.conf.get("cookie_ttl", 720))
-        domain = event["Records"][0]["cf"]["config"]["distributionDomainName"]
-        uri = event["Records"][0]["cf"]["request"]["uri"]
+        request = event["Records"][0]["cf"]["request"]
+        uri = request["uri"]
+        params = {k: v[0] for k, v in parse_qs(request["querystring"]).items()}
+        set_cookies = json.loads(cf_b64decode(params["CloudFront-Cookies"]))
```

Comment:
Can we please write this in a way so that it doesn't crash when there's a problem with this parameter?  (e.g. it's missing, not valid base64 or not valid JSON.)  We don't want to get lambda execution errors on bad inputs from clients because it'll make it hard to pick out or set up alerts for genuine errors from exodus-lambda.

It should probably also be validated that the JSON loading result strictly gives us a list of string and not anything else.

If any error does happen in this step, I think we should return a 400 (client error) response.